### PR TITLE
feat(TBD-7064): Integrate Spark batch to HDP Dynamic Distribution

### PR DESCRIPTION
**Pull request by zafkirTalend**
Relates to https://github.com/Talend/tbd-studio-ee/pull/597

### CODE REVIEW

**Comment by yu-talend** [comment]
Hi @cmeng-talend I added 2 module groups for teradata and oracle, but it is not able to load that jar finally in studio, so I did it by adding the jar in xml file at this moment. Do you have an idea about what's is going wrong ?
Thx.

File: main/plugins/org.talend.hadoop.distribution.hdpx/resources/template/hdp2xx/hdp2xxTemplate.json [hunk 1]
```diff
@@ -522,25 +541,109 @@
...
+			"useStudioRepository": "true",
+			"description": ""
 		}
 	],
 	"moduleGroups": [
 		{
-			"id": "EXAMPLE-SPARK-RELATED-MODULE-GROUP-DYNAMIC",
+			"id": "SPARK-LIB-SPARK2-DYNAMIC",
 			"description": "",
-			"supportedSparkVersions": [
```
[the remark this answers is not in the document]
Comment (yu-talend) [reply main/plugins/org.talend.hadoop.distribution.hdpx/resources/template/hdp2xx/hdp2xxTemplate.json]: @cmeng-talend idea ?

[the remark this answers is not in the document]
Comment (cmeng-talend) [reply main/plugins/org.talend.hadoop.distribution.hdpx/resources/template/hdp2xx/hdp2xxTemplate.json]: No idea, we need this property, seems currently the template only works with spark2?

**Comment by cmeng-talend** [comment]
Hello @yu-talend , about the problem related to module groups for teradata and oracle, can you please give me an example? so that I can reproduce it on my side and try to find the reason and fix it.

**Pull request merged**